Doc/library/fcntl.rst

```diff
@@ -176,17 +176,28 @@ The module defines the following functions:
    method are accepted as well) of the file to lock or unlock, and *cmd*
    is one of the following values:
 
-   * :const:`LOCK_UN` -- unlock
-   * :const:`LOCK_SH` -- acquire a shared lock
-   * :const:`LOCK_EX` -- acquire an exclusive lock
+   .. data:: LOCK_UN
 
-   When *cmd* is :const:`LOCK_SH` or :const:`LOCK_EX`, it can also be
-   bitwise ORed with :const:`LOCK_NB` to avoid blocking on lock acquisition.
-   If :const:`LOCK_NB` is used and the lock cannot be acquired, an
+             release an existing lock
+
+   .. data:: LOCK_SH
+
+             acquire a shared lock
+
+   .. data:: LOCK_EX
+
+             acquire an exclusive lock
```

Comment:
```suggestion
      Release an existing lock.

   .. data:: LOCK_SH

      Acquire a shared lock.

   .. data:: LOCK_EX

      Acquire an exclusive lock.
```

Fix formatting per [Style Guide](https://devguide.python.org/documentation/style-guide/#use-of-whitespace) and standard convention.